frontend/cypress/integration/admin/pool-candidate.spec.js

```diff
@@ -20,93 +20,118 @@ describe("Pool Candidates", () => {
       aliasQuery(req, "GetPoolCandidatesPaginated");
 
       aliasMutation(req, "UpdatePoolCandidateStatus");
+
+      // Only used if FEATURE_APPLICANTSEARCH is false
+      aliasQuery(req, "getUpdatePoolCandidateData");
+      aliasMutation(req, "updatePoolCandidate");
     });
 
     loginAndGoToPoolsPage();
   });
 
-  it("should update pool candidate status", () => {
-    cy.wait("@gqlgetPoolsQuery");
+  if (Cypress.env("FEATURE_APPLICANTSEARCH")) {
```

Comment:
Just a heads up to others that adding `?w=1` to the end of any diff-related url allows for review of diffs that ignore whitepace (just like how `-w` works on CLI).

https://github.com/GCTC-NTGC/gc-digital-talent/pull/4209/files?w=1